json-node/src/main/java/io/avaje/json/node/adapter/BooleanAdapter.java

```diff
@@ -0,0 +1,19 @@
+package io.avaje.json.node.adapter;
+
+import io.avaje.json.JsonAdapter;
+import io.avaje.json.JsonReader;
+import io.avaje.json.JsonWriter;
+import io.avaje.json.node.JsonBoolean;
+
+final class BooleanAdapter implements JsonAdapter<JsonBoolean> {
```

Comment:
so like why duplicate all of these adapters?

Reply:
Technically they are not dups.  Boolean is not JsonBoolean etc.

Reply:
I think your question is perhaps why have JsonNode at all?  ... ultimately I think that depends on the "helper" methods / functionality that JsonNode provides over "vanilla" types like List, Map, Boolean etc.

Or perhaps I misunderstand your question.

Reply:
like if they do the same thing as the adapters in the main jsonb, why not just export the adaters in core to this new module

Reply:
I see I have misread